README.md

```diff
@@ -7,7 +7,7 @@ This program generates a custom OpenTelemetry Collector binary based on a given
 $ GO111MODULE=on go get github.com/open-telemetry/opentelemetry-collector-builder
 $ cat > ~/.otelcol-builder.yaml <<EOF
 exporters:
-  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.21.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.22.0"
```

Comment:
Please, leave the version bumps to another commit.

Reply:
Yes, I thought I'd branched cleanly .. obviously not. I'll revert the version bump changes from this pull request